code/modules/vehicles/mecha/equipment/weapons/weapons.dm

```diff
@@ -202,9 +203,9 @@
 			if (istype(thegun, /obj/item/mecha_parts/mecha_equipment/weapon/energy/thermal/inferno))
 				thegun.equip_cooldown = 8
 		equip_cooldown = 8
-	return ..()
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/thermal/inferno/try_attach_part(mob/user, obj/vehicle/sealed/mecha/themech, attach_right)
+	. = ..()
```

Comment:
Line 213 has a `return FALSE` that needs to be updated

But also, moving the `..()` here changes behavior, no?

Before, it could early `return FALSE` to block attachment; now it can't do that

Reply:
You're right, oops